SPECS-SIGNED/systemd-boot-signed/systemd-boot-signed.spec

```diff
@@ -90,6 +93,9 @@ popd
 /boot/efi/EFI/BOOT/grubx64.efi
 
 %changelog
+* Thu Dec 12 2024 Daniel McIlvaney <damcilva@microsoft.com> - 255-19
+- Version bump to force signing with new Azure Linux secure boot key
```

Comment:
You're also adding package Recommends & Conflicts to ensure proper upgrade behavior. Please add a changelog note about this.

Reply:
```suggestion
- Version bump to force signing with new Azure Linux secure boot key
- Add confilcts/recommends on shim to ensure the keys match
```